### ISSUE TITLE
Umbilical cord with 4 vessels

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4737]
**Preferred term label:**
Umbilical cord with 4 vessels

**Synonyms**

**Definition (free text, please give PubMed ID)**
todo. Note normal is 3

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/pubmed/21391749

**Comment by pnrobinson** [comment]
adding as HP_0032513, @MickeySegal
